week2/starter/ColorPicker/LandScapeLayout.swift

```diff
@@ -0,0 +1,84 @@
+
+import SwiftUI
+
+struct LandScapeLayout: View {
+
+  @State private var redValue: Double = 250
+  @State private var greenValue: Double = 100
+  @State private var blueValue: Double = 50
+
+  @State private var committedRed: Double = 250
+  @State private var committedGreen: Double = 100
+  @State private var committedBlue: Double = 50
+
+  @Environment(\.horizontalSizeClass) var horizontalSizeClass
+  @Environment(\.verticalSizeClass) var verticalSizeClass
+
+
+    var body: some View {
+      ZStack {
+        HStack() {
+                   VStack {
+                       Text("Color Picker")
+                           .font(.largeTitle)
+                           .bold()
+                           .padding(.top, 10)
+                       RoundedRectangle(cornerRadius: 0)
+                           .fill(Color(red: committedRed/255, green: committedGreen/255, blue: committedBlue/255))
+                           .background(
+                               RoundedRectangle(cornerRadius: 0)
+                                   .strokeBorder(LinearGradient(gradient: Gradient(colors: [Color(red: committedRed/255, green: committedGreen/255, blue: committedBlue/255)]), startPoint: .leading, endPoint: .trailing),
+                                                 lineWidth: 10)
+                                   .blur(radius: 5)
+                           )
+                           .frame(minWidth: 250, idealWidth: 300, maxWidth: .infinity, minHeight: 200, idealHeight: 250, maxHeight: .infinity)
+                   }
+          VStack(spacing: 35) {
+                       colorSlider(title: "Red", value: $redValue, color: .red)
+                       colorSlider(title: "Green", value: $greenValue, color: .green)
```

Comment:
This was a unique implementation. I am curious why you didn't use a struct for this. Really cool to see you use a function that returns a some View. Curtis has done the same thing.